tests/Type/EnumTypeTest.php

```diff
@@ -655,4 +655,57 @@ public function testCallsOverwrittenEnumTypeMethods(): void
             GraphQL::executeQuery($this->schema, $query, null, null, $variables)->toArray(DebugFlag::INCLUDE_DEBUG_MESSAGE)
         );
     }
+
+    public function testLazilyDefineValuesAsCallable(): void
+    {
+        $called = false;
```

Comment:
```suggestion
        $called = 0;
```

Let's have the test case use the enum in both argument and result and assert this is only called once.